xbmc/network/httprequesthandler/python/HTTPPythonWsgiInvoker.cpp

```diff
@@ -351,8 +353,11 @@ std::map<std::string, std::string> CHTTPPythonWsgiInvoker::createCgiEnvironment(
   environment.insert(std::make_pair("PATH_INFO", pathInfo));
 
   // QUERY_STRING
-  CURL url(httpRequest->url);
-  environment.insert(std::make_pair("QUERY_STRING", url.GetOptions()));
```

Comment:
Why is `url.GetOptions()` not working anymore?

Reply:
Don't know if it ever worked in this case.

The full url is not in `httpRequest->url`, only the path + options. 
In that case, CURL does not parse the options (https://github.com/xbmc/xbmc/blob/master/xbmc/URL.cpp#L80), so `url.GetOptions()` is empty

Reply:
Any objection @Montellese ?